heft-plugins/heft-storybook-plugin/src/StorybookPlugin.ts

```diff
@@ -117,14 +130,35 @@ export class StorybookPlugin implements IHeftPlugin<IStorybookPluginOptions> {
         '(EXPERIMENTAL) Used by the "@rushstack/heft-storybook-plugin" package to launch Storybook.'
     });
 
+    const storybookStaticBuildParameters: IHeftFlagParameter = heftSession.commandLine.registerFlagParameter({
+      associatedActionNames: ['build'],
+      parameterLongName: '--storybook',
+      description:
+        '(EXPERIMENTAL) Used by the "@rushstack/heft-storybook-plugin" package to start static build Storybook.'
+    });
+
     heftSession.hooks.build.tap(PLUGIN_NAME, (build: IBuildStageContext) => {
-      if (!storybookParameters.actionAssociated || !storybookParameters.value) {
+      if (
+        (!storybookParameters.actionAssociated || !storybookParameters.value) &&
+        (!storybookStaticBuildParameters.actionAssociated || !storybookStaticBuildParameters.value)
+      ) {
         this._logger.terminal.writeVerboseLine(
           'The command line does not include "--storybook", so bundling will proceed without Storybook'
         );
         return;
       }
 
+      const modulePath: string | undefined = storybookStaticBuildParameters.actionAssociated
+        ? options.staticBuildModulePath
+        : options.startupModulePath;
+      if (!modulePath) {
```

Comment:
This check is now duplicated, since you are already checking on line 119

Reply:
This check for if the user only specify `startupModulePath` in the heft.json, but run it with `heft build --storybook` command. Then it will pass the check on line 119, but no static build path.

Reply:
Ah yes, you're right